src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -82,8 +80,10 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     case CallOutputs(workflowId, callName) => callOutputs(workflowId, callName) pipeTo sender
     case CallStdoutStderr(workflowId, callName) => callStdoutStderr(workflowId, callName) pipeTo sender
     case WorkflowStdoutStderr(workflowId) => workflowStdoutStderr(workflowId) pipeTo sender
-    case CurrentState(actor, state: WorkflowState) => updateWorkflowState(actor, state)
-    case Transition(actor, oldState, newState: WorkflowState) => updateWorkflowState(actor, newState)
+    case Transition(actor, oldState, newState: WorkflowState) =>
+      updateWorkflowState(actor, newState)
+      // This is hokey but WorkflowActor can't shut itself down due to tests relying on it lasting past completion
```

Comment:
IIRC it was everything handling WorkflowActor. And as a hakker would say they're all integration tests. The issue is that they set up a workflow actor, run something and then ask it for its outputs

I'm pretty sure that craptastic only-for-testing code path used to be used for other things and now no one gets rid of it